src/main/scala/org/deepdive/ddlog/DeepDiveLogSemanticChecker.scala

```diff
@@ -156,6 +157,46 @@ object DeepDiveLogSemanticChecker extends DeepDiveLogHandler {
     }
   }
 
+  def checkSupervisionLableType(s: ExtractionRule, expType: VariableType, SupVariable: VarPattern, b: BodyAtom) {
+    b.terms.zipWithIndex.foreach {
+      case (SupVariable, index) => {
+        val expColumnType = if (expType.isInstanceOf[BooleanType.type]) "boolean" else "text"
```

Comment:
I guessed it from test/expected-output-test/chunking_example/input.ddlog, which uses Categorial supervision column. Let me know if my assumption is wrong :)